test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegateConsolidation.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env mayapy
+#
+# Copyright 2021 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import fixturesUtils
+import imageUtils
+import mayaUtils
+import usdUtils
+import testUtils
+
+from mayaUsd import lib as mayaUsdLib
+from mayaUsd import ufe as mayaUsdUfe
+
+from maya import cmds
+
+from pxr import Vt
+
+import ufe
+
+import os
+
+
+class testVP2RenderDelegateConsolidation(imageUtils.ImageDiffingTestCase):
```

Comment:
Just double-checking, but this test is valid for any Maya version with UFE >= 2 ?

Reply:
Yes I believe so. There is only a relatively minor dependency on UFE to call getStage. I use USD to change the colors of the materials.